Directory.Packages.props

```diff
@@ -13,6 +13,7 @@
 
     <!-- These versions are not updated by dependency flow as they aren't produced live anymore. -->
     <PackageVersion Include="System.Numerics.Vectors" Version="4.5.0" />
+    <PackageVersion Include="System.Reflection.Emit" Version="4.7.0" />
```

Comment:
I used the latest available version found in nuget: https://www.nuget.org/packages/System.Reflection.Emit